frontend/components/global/AutoForm.vue

```diff
@@ -37,15 +37,16 @@
           :name="inputField.varName"
           :disabled="(inputField.disableUpdate && updateMode) || (!updateMode && inputField.disableCreate) || (disabledFields && disabledFields.includes(inputField.varName))"
           :hint="inputField.hint"
-          hide-details="auto"
+          :hide-details="!inputField.hint"
+          :persistent-hint="!!inputField.hint"
```

Comment:
We use the hint as the checkbox description, so if we have one, we should always display it